config/pipeline.yaml

```diff
@@ -449,6 +527,29 @@ jobs:
       - 'kernelci'
       - 'stable-rc'
 
+  kbuild-gcc-12-arc-mainline: &kbuild-gcc-12-arc-mainline-job
+    <<: *kbuild-gcc-12-arc-job
+    params: &kbuild-gcc-12-arc-mainline-params
+      <<: *kbuild-gcc-12-arc-params
+      fragments:
+        - 'debug'
+        - 'kselftest'
+        - 'tinyconfig'
+    rules:
```

Comment:
These rules have been translated from
```
            - blocklist:
                defconfig:
                  - 'axs101_defconfig'
                  - 'nps_defconfig'
                  - 'nsim_700_defconfig'
                  - 'nsimosci_defconfig'
                  - 'tb10x_defconfig'
            # haps_hs_smp_defconfig not available in some old kernels
                kernel: ['v3.', 'v4.4', 'v4.9']
```
Please confirm if I've translated correctly:
- I think we don't need to add the rule to exclude the mentioned configs as we aren't enabling them.
- I've enabled builds for kernel versions starting from 4.10.

Reply:
Yes, correct